easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.1.1-goolf-1.7.20.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'Xerces-C++'
+version = '3.1.1'
+
+homepage = 'http://xerces.apache.org/xerces-c/'
+description = """Xerces-C++ is a validating XML parser written in a portable 
+ subset of C++. Xerces-C++ makes it easy to give your application the ability 
+ to read and write XML data. A shared library is provided for parsing, generating, 
+ manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['http://apache.belnet.be/xerces/c/3/sources/']
+sources = ['xerces-c-%(version)s.tar.gz']
+
+dependencies = [
+    ('cURL', '7.40.0'),
+    #('cURL', '7.40.0', '', True), # Compile Curl with dummy toolchain (system gcc)
```

Comment:
drop this line (when we have support for subtoolchains implemented, you won't need to change this anymore)